operations/bills.md

```diff
@@ -180,6 +180,7 @@ Returns all bills, optionally filtered by customers, identifiers and other filte
 | `VariableSymbol` | string | optional | Variable symbol of the bill. |
 | `CreatedUtc` | string | required | Date and time of the bill creation in UTC timezone in ISO 8601 format. |
 | `IssuedUtc` | string | required | Date and time of the bill issuance in UTC timezone in ISO 8601 format. |
+| `UpdatedUtc` | string | | |
```

Comment:
```suggestion
| `UpdatedUtc` | string | optional | Date and time when the bill was last updated, in UTC timezone in ISO 8601 format. |
```

Reply:
I can confirm that UpdatedUtc is returned 👍